plugins/net_plugin/net_plugin.cpp

```diff
@@ -2683,6 +2684,10 @@ namespace eosio {
          char reason[72];
          snprintf(reason, 72, "Dropping trx, too many trx in progress %lu bytes", trx_in_progress_sz);
          my_impl->producer_plug->log_failed_transaction(ptr->id(), ptr, reason);
+         if (fc::time_point::now() - fc::seconds(1) >= last_dropped_trx_msg_time) {
+            last_dropped_trx_msg_time = fc::time_point::now();
+            wlog("Speculative execution is REJECTING transactions, too many trx in progress");
```

Comment:
Remove "Speculative execution is" as this is not necessarily speculative execution. Also, go ahead and log the `trx_in_progress_sz`. For that matter, might as well just `wlog(reason);`